android/app/buildConfigs.gradle

```diff
@@ -48,16 +59,13 @@ def setupGoogleServices(buildConfig, resValue) {
 }
 
 def setupResourceValues(buildConfigName, resValue) {
+
     def command = "yarn --silent babel-node tools/build-config to-xcconfig $buildConfigName"
-    def proc = command.execute()
-    proc.waitFor()
+    def proc = execCommand(command)
 
-    if (proc.exitValue() != 0) {
-        println("Failed to get xcconfig file: $proc.err.text")
-        return
-    }
```

Comment:
Why did you remove this? This is kind of important for debugging if the execution of the command fails.

Reply:
This has caused the last CI failure. Re-entered with re-arrangement as next.

Reply:
Ahh yes I guess I know what the problem is now. `proc.in.text` waits until the process is finished and the input stream is closed. If you call `proc.exitValue()` before that and without `proc.waitFor()`, the process is not yet finished and fails therefore. So you probably have to move this block after `proc.in.text`.

Reply:
Still relevant.

Reply:
Still relevant :)